src/NuGet.Services.EndToEnd/Support/Package.cs

```diff
@@ -100,5 +135,19 @@ public static Package SignedPackage()
 
             return new Package(id, normalizedVersion, fullVersion, nupkgBytes);
         }
+
+        private static MemoryStream GetMemoryStreamForFile(string filename)
+        {
+            var memoryStream = new MemoryStream();
+            using (FileStream fileStream = File.OpenRead(filename))
+            {
+                memoryStream.SetLength(fileStream.Length);
+                fileStream.Read(memoryStream.GetBuffer(), 0, (int)fileStream.Length);
```

Comment:
Can you use [`Stream.CopyTo`]([https://docs.microsoft.com/en-us/dotnet/api/system.io.stream.copyto?view=netframework-4.7.2#System_IO_Stream_CopyTo_System_IO_Stream](https://docs.microsoft.com/en-us/dotnet/api/system.io.stream.copyto?view=netframework-4.7.2#System_IO_Stream_CopyTo_System_IO_Stream)_) instead?